### PR TITLE
Add ScheduleMethod strategies

### PR DESCRIPTION
**Pull request by slavcodev**

### CODE REVIEW

File: spec/definitions/Scheduling/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Scheduling objects
+
+## Objects
+
+### Schedule method
+
+Strategies how to calculate relative time. It answer on question "when to schedule"
```
Comment (adamaltman) [inline spec/definitions/Scheduling/README.md]: `It answers the question of "when" to schedule.`

**Comment by vova07** [comment]
What about `event-time` strategy ?

File: spec/definitions/Scheduling/README.md (hunk 1 above)
Comment (slavcodev) [inline spec/definitions/Scheduling/README.md]: Thanks.

**Comment by slavcodev** [comment]
> What about event-time strategy ?

That not answer on "when", it answer the question "what is base time"

**Pull request merged**